README.md

```diff
@@ -97,6 +97,14 @@ explicitly assumed by this plugin).
 This should refer to a stack configuration file that exists within the
 specified `project_dir` (see above).
 
+### stack (optional, string)
```

Comment:
Copy/paste error; this should be `refresh (optional, boolean)`

Reply:
asdfdsafdsaf